pkg/query-service/cache/inmemory/options.go

```diff
@@ -0,0 +1,21 @@
+package inmemory
+
+import (
+	"time"
+)
+
+const (
+	defaultTTL             = 5 * time.Minute
```

Comment:
Shouldn't default ttl be forever with no cleanup required?

Reply:
I was thinking not to let this grow forever and cause memory issues, but I see where you are coming from. If I understand you correctly, the default should be set to the maximum possible, but the user who creates the cache instances should pass what's appropriate for them, is that correct?

Reply:
Yeah, that's where I was coming form. Though any serious user will set the values explicitly.

Reply:
Updated